### PR TITLE
Rnp 72

### PR DESCRIPTION
**Pull request by robertszuba**

### CODE REVIEW

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt [hunk 1]
```diff
@@ -3,40 +3,79 @@ package com.onegini.mobile.sdk.reactnative.managers
 import com.onegini.mobile.sdk.reactnative.OneginiSDK
 import com.onegini.mobile.sdk.reactnative.exception.EmptyOneginiErrorDetails
```
Comment (aleknow) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt:4]: Unused imports

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt [hunk 2]
```diff
@@ -3,40 +3,79 @@ package com.onegini.mobile.sdk.reactnative.managers
...
-        oneginiSDK.oneginiClient.userClient.registerAuthenticator(authenticator, handler)
+        oneginiSDK.oneginiClient.userClient.registerAuthenticator(
+            authenticator,
+            object : OneginiAuthenticatorRegistrationHandler {
+                override fun onSuccess(info: CustomInfo?) {
+                    callback.onSuccess(info)
+                }
+
+                override fun onError(error: OneginiAuthenticatorRegistrationError) {
+                    callback.onError(error?.errorType.toString(), error?.message)
```
Comment (aleknow) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt:40]: Unnecessary safe call on `error` object

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt [hunk 3]
```diff
@@ -3,40 +3,79 @@ package com.onegini.mobile.sdk.reactnative.managers
...
-        oneginiSDK.oneginiClient.userClient.deregisterAuthenticator(authenticator, handler)
+        oneginiSDK.oneginiClient.userClient.deregisterAuthenticator(
+            authenticator,
+            object : OneginiAuthenticatorDeregistrationHandler {
+                override fun onSuccess() {
+                    callback.onSuccess()
+                }
+
+                override fun onError(error: OneginiAuthenticatorDeregistrationError) {
+                    callback.onError(error?.errorType.toString(), error?.message)
```
Comment (aleknow) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt:67]: Unnecessary safe call on `error` object

**Review by aleknow — CHANGES_REQUESTED** [review]

**Comment by robertszuba** [comment]
All imports should be fixed now

**Review by aleknow — APPROVED** [review]

**Pull request merged**